iai_kitchen/urdf/sink_area.urdf.xacro

```diff
@@ -19,6 +19,12 @@
           <box size=".1 .1 .1" />
         </geometry>
       </visual>
+      <collision>
```

Comment:
@artnie Please get rid of this collision object. It's a virtual link, we don't need a collision object for it.